src/odemis/acq/stitching/test/registrar_test.py

```diff
@@ -311,46 +301,61 @@ class TestShiftRegistrar(unittest.TestCase):
 
     def setUp(self):
         random.seed(1)
+        numpy.random.seed(10)
```

Comment:
I think you don't need anything specific from numpy.random. In this case, you could drop this line (and its clone in the other test case).

Reply:
thanks, good catch.